webpack.config.js

```diff
@@ -0,0 +1,78 @@
+var webpack = require('webpack');
+var path = require('path');
+var NODE_ENV = process.env.NODE_ENV === 'Development';
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const extractSass = new ExtractTextPlugin({
+    filename: '[name].[contenthash].css',
+    disable: process.env.NODE_ENV === 'Development'
+});
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
+
+module.exports = {
+    context: __dirname,
+    entry: {
+        app: 'react-hot-loader/patch',
+        bundle: './src/index.js',
+        styles: './src/index.scss'
+    },
+    output: {
+        filename: '[name].js',
+        path: path.resolve(__dirname, './build')
+    },
+    resolve: {
+        extensions: ['.js', '.jsx']
+    },
+    module: {
+        rules: [
+            {
+                test: /\.jsx?$/,
+                exclude: /(node_modules|bower_components)/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['react', 'es2015'],
+                        plugins: [
+                            'react-hot-loader/babel'
+                        ]
+                    }
+                }
+            },
+            {
+                test: /\.scss$/,
+                use: extractSass.extract({
+                    use: [{
+                        loader: 'css-loader'
+                    }, {
+                        loader: 'sass-loader'
+                    }],
+                    fallback: 'style-loader'
+                })
+            },
+            {
+                test: /\.svg$/,
+                loader: 'svg-inline-loader'
+            }
+        ]
+    },
+    plugins: [
+        new ExtractTextPlugin('styles.css', {
+            allChunks: true
+        })
+    ],
+    devtool: (NODE_ENV ? 'source-map' : false),
+    devServer: {
+        contentBase: path.join(__dirname, 'build'),
+        compress: true,
+        port: 9000
+    }
+};
+
+if (!NODE_ENV) {
+    module.exports.plugins.push(new UglifyJSPlugin({
```

Comment:
Create a variable, like `config`, and assign base of you webpack config to it. Then, change some params according to NODE_ENV, and only after config is ready export it. It is strange to change `module.exports` on the fly.